.github/SUPPORT.md

```diff
@@ -0,0 +1,10 @@
+
+# Looking for support? We want to help
+
+Please post your question to X (#) using the Timber tag.
+
+## Other places
+
+You can use [GitHub Discussions](#) to open up questions and engage in discussions with the contributors and other users.
```

Comment:
Do we plan to use GitHub Discussion? That's fine. I'm just curious. 😄

Reply:
I asked the frontend team, and for the moment, we'll stick with Zulip.